site/public/js/grade-inquiry.js

```diff
@@ -1,6 +1,27 @@
 /* global buildCourseUrl, WebSocketClient */
 /* exported initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
 
+function saveDraft() { 
+    //saveDraft function displays the saved draft text on grade inquiry box
+    let draftContent = localStorage.getItem('content');
+    if (draftContent === null) {
+        draftContent = {};
+    }
+    else {
+        draftContent = JSON.parse(draftContent);
+    }
+    const elements = document.getElementsByClassName('markdown-textarea fill-available ');
+    for (let i = 0; i < elements.length; i++) {
+        const element = elements[i];
+        const elementId = element.getAttribute('id');
+        const reply_text_area = $(element);
+        if (draftContent.hasOwnProperty(elementId)){
```

Comment:
```suggestion
        if (Object.prototype.hasOwnProperty.call(draftContent, elementId)) {
```
Change this to use `Object.prototype.hasOwnProperty.call`, because for whatever reason that is what the linter wants.